pages/k8s/cni-canal.md

```diff
@@ -0,0 +1,72 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "CNI with canal"
+  description: How to manage and deploy Kubernetes with Canal
+keywords: CNI, networking
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: cni-canal.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+'Canal' is a shorthand for saying "Calico and Flannel", a common practise which sets up Calico to handle policy management and Flannel to manage the network itself. The manual setup is explained in the [Calico documentation][canal].  
+
+
+## Deploying Charmed Kubernetes with Canal
+
+To deploy CDK with Canal, deploy the kubernetes-canal bundle:
+
+```bash
+juju deploy cs:bundle/canonical-kubernetes-canal
+```
+
+The canal bundle is identical to the standard `charmed-kubernetes` bundle with the
+exception of replacing flannel with canal. You can apply any customisation overlays
+that would apply to `charmed-kubernetes` to this bundle also.
+
+## Canal configuration options
+
+|Name                  | Type    | Default   | Description                      |
+|======================|=========|===========|==================================|
+| calico-node-image    | string  | quay.io/calico/node:v2.6.12|The image id to use for calico/node. |
+| calico-policy-image  | string  | quay.io/calico/kube-controllers:v1.0.5|The image id to use for calico/kube-controllers. |
```

Comment:
These defaults have changed since this was written:

calico-node-image: `docker.io/calico/node:v3.6.1`
calico-policy-image: `docker.io/calico/kube-controllers:v3.6.1`